argo-cd-apps/overlays/development/kustomization.yaml

```diff
@@ -12,7 +12,6 @@ resources:
   - ../../base/quality-dashboard
```

Comment:
should we do the same for quality-dashboard?

Reply:
yes, I will do a separate PR for it

Reply:
https://github.com/redhat-appstudio/infra-deployments/pull/3702